config/lint.xml

```diff
@@ -57,4 +57,6 @@
     <!-- This check crashes if request interval is passed as a param :S -->
     <issue id="InvalidPeriodicWorkRequestInterval" severity="ignore" />
     <issue id="AlwaysShowAction" severity="ignore" />
+
+    <issue id="InvalidFragmentVersionForActivityResult" severity="ignore" />
```

Comment:
Lint check fails. See https://github.com/getodk/collect/pull/4660/commits/809b312d5ca9a43bb6d51275dd915c16597e02c3, looks like a false positive to me. Nothing would work otherwise...